discord-bot/src/commands/poll/index.ts

```diff
@@ -52,14 +73,11 @@ const createPoll = async (msg: Message) => {
   }
 
   const currentTextChannel = channel as TextChannel;
-
-  let webhook = await fetchWebhook(currentTextChannel);
-
-  if (!webhook) {
-    webhook = await createWebhook(currentTextChannel); // create webhook if not found
-  }
-
-  if (!webhook) return; // return if can't find or create webhook
+  const webhook = await fetchOrCreateWebhook(
+    currentTextChannel,
+    'VAIT-Poll-Hook'
```

Comment:
should we use a generic name for our webhook? Individual webhook for each function is redundant.